build/Targets/Settings.targets

```diff
@@ -135,6 +135,8 @@
   <PropertyGroup>
     <IsVSBeforeDev15Preview4 Condition="'$(OS)' == 'Windows_NT' AND '$(registry:HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\VisualStudio\15.0@InstallDir)' != ''">true</IsVSBeforeDev15Preview4>
     <IsVSBeforeDev15Preview4 Condition="'$(OS)' == 'Windows_NT' AND '$(registry:HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\Microsoft\VisualStudio\15.0@InstallDir)' != ''">true</IsVSBeforeDev15Preview4>
+    <IsVSBeforeDev15Preview4 Condition="'$(IsVSBeforeDev15Preview4)' == 'true' AND Exists('$(MSBuildBinPath)\..\..\..\Common7\IDE\devenv.isolation.ini')">false</IsVSBeforeDev15Preview4>
```

Comment:
We probably shouldn't be dependent on it, especially if we ever want to Roslyn to be buildable without VS installed.

Everything VS related should be coming from a NuGet package and this temporary hack should go away once we RTM.

However, I can definitely switch it, if you feel that it would make the hack any better :smile: